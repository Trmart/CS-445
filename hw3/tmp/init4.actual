.   .   .   .   Child: 0  Const 23 [line: 3]
.   .   .   Child: 0  Id: x [line: 4]
.   .   .   Child: 0  Op: * [line: 3]
.   .   Child: 0  Var: x of type int [line: 3]
.   Child: 1  Compound [line: 2]
.   .   .   .   Child: 1  Const 3 [line: 3]
Func: main returns type int [line: 1]
.   .   Sibling: 1  Var: y of type int [line: 4]
.   .   Sibling: 2  Var: roll of type int [line: 5]
