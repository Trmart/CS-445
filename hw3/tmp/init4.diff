1,5c1,3
< .   .   .   .   Child: 0  Const 23 of type int [line: 3]
< .   .   .   .   Child: 0  Const 6 of type int [line: 5]
< .   .   .   Child: 0  Id: x of type int [line: 4]
< .   .   .   Child: 0  Op: * of type int [line: 3]
< .   .   .   Child: 0  Op: ? of type int [line: 5]
---
> .   .   .   .   Child: 0  Const 23 [line: 3]
> .   .   .   Child: 0  Id: x [line: 4]
> .   .   .   Child: 0  Op: * [line: 3]
8,9c6
< .   .   Child: 1  Const 333 of type int [line: 6]
< .   .   .   .   Child: 1  Const 3 of type int [line: 3]
---
> .   .   .   .   Child: 1  Const 3 [line: 3]
11,13d7
< Number of errors: 0
< Number of warnings: 2
< .   .   Sibling: 1  Const 'z' of type char [line: 7]
15d8
< .   .   Sibling: 2  Const true of type bool [line: 8]
17,18d9
< WARNING(4): The variable 'y' seems not to be used.
< WARNING(5): The variable 'roll' seems not to be used.
