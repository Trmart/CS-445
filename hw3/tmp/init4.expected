.   .   .   .   Child: 0  Const 23 of type int [line: 3]
.   .   .   .   Child: 0  Const 6 of type int [line: 5]
.   .   .   Child: 0  Id: x of type int [line: 4]
.   .   .   Child: 0  Op: * of type int [line: 3]
.   .   .   Child: 0  Op: ? of type int [line: 5]
.   .   Child: 0  Var: x of type int [line: 3]
.   Child: 1  Compound [line: 2]
.   .   Child: 1  Const 333 of type int [line: 6]
.   .   .   .   Child: 1  Const 3 of type int [line: 3]
Func: main returns type int [line: 1]
Number of errors: 0
Number of warnings: 2
.   .   Sibling: 1  Const 'z' of type char [line: 7]
.   .   Sibling: 1  Var: y of type int [line: 4]
.   .   Sibling: 2  Const true of type bool [line: 8]
.   .   Sibling: 2  Var: roll of type int [line: 5]
WARNING(4): The variable 'y' seems not to be used.
WARNING(5): The variable 'roll' seems not to be used.
